### ISSUE TITLE
Bug:  Audio out not switching to bluetooth headphones on Mac

### ISSUE THREAD

**Issue by dheista** [Grix/ildagen#13]
When I have my airpods pro in my ears, the Mac automatically recognizes them, and I can play Youtube, Apple Music or other applications and hear the audio through the bluetooth headphones.  
But when I play in Timeline mode on LSG, the audio is still forced through the laptop speakers.
I checked the audio/sound setting on the Mac and it does show the audio output is through the headphones (as shown in the attached screenshot).

<img width="648" alt="Screen Shot 2021-12-27 at 2 09 05 PM" src="https://user-images.githubusercontent.com/1724371/147500408-c8531f7a-0ca3-4061-8b3b-6c5f69a1a828.png">

**Comment by dheista** [comment]
BTW - this is the M1 Macbook Pro, running OS X Monterrey.  I haven't tried on my Mac Pro at home running Catalina yet.

**Comment by dheista** [comment]
Hmm - after doing a little digging, I found I needed to enable the airpods with the LSG application CLOSED, and then open LSG.  Only then does it recognize the changed audio output device.  Found on this thread: 
https://www.reddit.com/r/gpdwin/comments/b8sovz/gpd_win_2_and_airpods_2_no_sound_in_games/

Guess you can close this bug, unless you want to dig deeper and figure out how to probe the audio output settings on some schedule or trigger.

**Comment by Grix** [comment]
Indeed LSG opens the audio system on startup, with the default audio device. So the audio device needs to be set up when LSG is started. I can keep the issue open but this probably won't get changed any time soon since it's a bit of an involved process, having to modify an external library instead of just the main program.

**Comment by dheista** [comment]
Yeah - this is probably just fine since I figured out it has to have control of the device before opening LSG.  Maybe just an additional FAQ entry, so people can find a solution to the problem if they don't have a problem-solving kinda brain (which I'm not sure a non-problem-solving brain would be interested in LSG or even lasers!). I think it's safe to close this one.
